src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -71,28 +78,11 @@ object HttpGoogleServicesDAO {
     if ( actualState != expectedState ) throw new OAuthException(
       "State mismatch: this authentication request cannot be completed.")
 
-    val gcsTokenResponse: GoogleTokenResponse = flow.newTokenRequest(authCode)
+    val gcsTokenResponse = flow.newTokenRequest(authCode)
       .setRedirectUri(callbackUri.toString)
       .execute()
 
-    val refreshToken = gcsTokenResponse.getRefreshToken match {
```

Comment:
This is all reformatting Google's token format to ours, so I felt it belonged with our token format's class instead of here